resources/zib/zib-Payer.xml

```diff
@@ -0,0 +1,189 @@
+        </element>
+        <element id="Coverage.type">
+            <path value="Coverage.type"/>
+            <short value="InsuranceType"/>
+            <definition value="Type of insurance policy. Codes as returned in the Check for Right to Insurance"/>
+            <alias value="Verzekeringssoort"/>
+            <binding>
+                <strength value="required"/>
+                <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.1.1.1--20200901000000"/>
+            </binding>
+            <mapping>
+                <identity value="zib-payer-v3.1.1-2020EN"/>
+                <map value="NL-CM:1.1.15"/>
+                <comment value="InsuranceType"/>
+            </mapping>
+        </element>
+        <element id="Coverage.subscriberId">
+            <path value="Coverage.subscriberId"/>
+            <short value="InsurantNumber"/>
+            <definition value="Number under which the insured person is registered at the insurance company This item maps the ‘Identification number of the card’ on EHIC field 8&#xA;                        "/>
+            <alias value="VerzekerdeNummer"/>
+            <mapping>
+                <identity value="zib-payer-v3.1.1-2020EN"/>
+                <map value="NL-CM:1.1.6"/>
+                <comment value="InsurantNumber"/>
+            </mapping>
+        </element>
+        <element id="Coverage.beneficiary">
+            <path value="Coverage.beneficiary"/>
+            <type>
+                <code value="Reference"/>
+                <targetProfile value="http://hl7.org/fhir/StructureDefinition/Patient"/>
+                <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-Patient"/>
+            </type>
+        </element>
+        <element id="Coverage.period.start">
+            <path value="Coverage.period.start"/>
+            <short value="StartDateTime"/>
+            <definition value="Date from which the insurance policy coverage applies."/>
+            <alias value="BeginDatumTijd"/>
+            <mapping>
+                <identity value="zib-payer-v3.1.1-2020EN"/>
+                <map value="NL-CM:1.1.13"/>
+                <comment value="StartDateTime"/>
+            </mapping>
+        </element>
+        <element id="Coverage.period.end">
+            <path value="Coverage.period.end"/>
+            <short value="EndDateTime"/>
+            <definition value="Date until which the insurance policy coverage applies. This item maps the ‘Expiry date’ on EHIC field 9."/>
+            <alias value="EindDatumTijd"/>
+            <mapping>
+                <identity value="zib-payer-v3.1.1-2020EN"/>
+                <map value="NL-CM:1.1.14"/>
+                <comment value="EndDateTime"/>
+            </mapping>
+        </element>
+        <element id="Coverage.payor">
+            <path value="Coverage.payor"/>
+            <slicing>
+                <discriminator>
+                    <type value="profile"/>
```

Comment:
Can't this slicing be handled based on type = _type_ ? You don't need to check conformance to a profile to differentiate between Patient/RelatedPerson and Organization.

Reply:
See above